### PR TITLE
add pointers for manual testing

### PR DESCRIPTION
**Pull request by sed-i**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -83,6 +83,16 @@ $ tox -e unit
 
 It is expected that unit tests should provide at least 80% code coverage.
 
+### Manual testing
```
Comment (balbirthomas) [inline CONTRIBUTING.md:86]: I do not have strong opinions here but I do not think it makes much sense to provide instructions on how to use  Prometheus or its API in `CONTRIBUTING.md`. I would expect anyone who is going to make a PR against this repo has spent the time to study the [Prometheus docs](https://prometheus.io/docs/) well enough.

Comment (sed-i) [reply CONTRIBUTING.md:86]: I agree, I would put this in README or TESTING (if we had one). However, the "Testing" section is in CONTRIBUTING.
That being said, even after being familiar with the docs, it is not immediately obvious how to quickly probe the endpoints.

Comment (balbirthomas) [reply CONTRIBUTING.md:86]: [`src/prometheus_server.py`](https://github.com/canonical/prometheus-operator/blob/main/src/prometheus_server.py#L56) already has an example of using the Prometheus API. Anyone working on this charm should know how to query the Prometheus API. I see know point in re-documenting or exemplifying the use of the Prometheus API which is already documented.

Comment (sed-i) [reply CONTRIBUTING.md:86]: Should add `api/v1/status/buildinfo` to the list of "poorly documented endpoints"

Comment (dstathis) [reply CONTRIBUTING.md:86]: I agree, it seems like you are going to have to look up the Prometheus docs anyway so I don't really see the need for this.

**Comment by sed-i** [comment]
I captured the proposed changes in a [gist](https://gist.github.com/sed-i/76e846846deb2eff614c7aa771aa18a1). Closing.

**Pull request closed without merging**